utils/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use an official Python runtime as a parent image
+FROM ubuntu:latest
+
+ENV TZ=Asia/Singapore
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the requirements.txt into the container at /app
+#COPY [ "gtfTobed.py", "/app/" ]
+
+# Install any needed packages specified in requirements.txt
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential python3 python3-setuptools python3-dev python3-pip libcurl4
+RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
+RUN wget https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/gtfToGenePred
+RUN wget https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/genePredToBed
+RUN chmod +x gtfToGenePred genePredToBed
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+ENTRYPOINT ["python","gtfTobed.py"]
```

Comment:
In this paper https://f1000research.com/articles/7-742/v2 (recommendation 4.) they suggest avoiding ENTRYPOINT.  In general, I also think that "gtfTobed.py" should be made executable, such that the command in the workflows can then be written as `python gtfTobed.py` instead of `python app/gtfTobed.py`, as this might be confusing for someone who is just pulling the image, and then wouldn't intuitively know which path to provide here.

Reply:
I have removed the `ENTRYPOINT` line. Now the gtfTobed.py can be executed by `/gtfTobed.py` https://github.com/iRNA-COSI/APAeval/blob/5a2673ea3b5166174f90f793cba01bd5e511eaff/utils/nextflow_minitest/gtftobed.nf#L20